telcell/models/simplemodel.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import numpy as np
+from typing import List
+
+from telcell.data.models import Measurement, Track, MeasurementPair
+
+
+def get_measurement_with_minimum_time_difference(track: Track,
+                                                 timestamp: datetime) \
+        -> Measurement:
+    """
+    Finds the measurement in the track with the smallest time
+    difference compared to the timestamp.
+
+    @param track: A history of measurements for a single device.
+    @param timestamp: The timestamp used to find the closest measurement.
+    @return: The measurement that is the closest to the timestamp
+    """
+    track_a_times = [x.timestamp for x in track.measurements]
+    time_difference_with_timestamp = [abs(timestamp - x)
+                                      for x in track_a_times]
+    index = np.argmin(time_difference_with_timestamp)
+    return track.measurements[index]
+
+
+def make_pair_based_on_time_difference(track: Track,
+                                       measurement: Measurement) \
+        -> MeasurementPair:
+    """
+    Creates a pair based on time difference. The closest measurement in
+    absolute time will be paired to the measurement.
+
+    @param track: A history of measurements for a single device.
+    @param measurement: A single measurement of a device at a certain
+                        place and time.
+    @return: A pair of measurements.
+    """
+    closest_measurement = get_measurement_with_minimum_time_difference(
+        track,
+        measurement.timestamp)
+    return MeasurementPair(closest_measurement, measurement)
+
+
+def pair_measurements_based_on_time(track_a: Track,
+                                    track_b: Track) \
+        -> List[MeasurementPair]:
+    """
+    Pairs two tracks based on the time difference between the measurements.
```

Comment:
Wat toelichting over de asymmetrie tussen `track_a` en `track_b` zou fijn zijn hier denk ik. Immers: van elke measurement uit `track_b` wordt een `MeasurementPair` gemaakt, maar dat geldt niet voor elke measurement uit `track_a`, dat is mogelijk verwarrend.

Reply:
Aangepast naar:   Pairs two tracks based on the time difference between the measurements. 
    It pairs all measurements from track_b to the closest pair of track_a, 
    meaning that not all measurements from track_a have to be present in the 
    final list!